packages/octave/build.sh

```diff
@@ -48,12 +71,29 @@ termux_step_post_get_source() {
 termux_step_pre_configure() {
 	termux_setup_flang
 
+	# termux_setup_flang() sometimes runs cd "$FLANG_FOLDER_TMP" and then does not go back to original directory
+	cd "$TERMUX_PKG_SRCDIR"
+
 	local flang_toolchain_dir="$(dirname $(dirname $(command -v flang-new)))"
 	local flang_libs_dir="$flang_toolchain_dir/sysroot/usr/lib/$TERMUX_HOST_PLATFORM"
 
 	export F77="$FC"
 	export ac_cv_f77_libs=" $flang_libs_dir/libFortranRuntime.a $flang_libs_dir/libFortranDecimal.a"
 
+	host_clang_major_version=$($CXX --version | grep -m1 version | sed -E 's|.*\bclang version ([0-9]+).*|\1|')
```

Comment:
I actually don't think this a proper solution. It is better export `ac_cv_f77_libs` rather than patch it here.

(Untested, as I don't have a computer with termux-builder now)

```patch
-export ac_cv_f77_libs=" $flang_libs_dir/libFortranRuntime.a $flang_libs_dir/libFortranDecimal.a"
+mkdir -p $TERMUX_PKG_TMPDIR/_deps
+ln -sf $flang_libs_dir/libFortranRuntime.a $TERMUX_PKG_TMPDIR/_deps/
+ln -sf $flang_libs_dir/libFortranDecimal.a $TERMUX_PKG_TMPDIR/_deps/
+export ac_cv_f77_libs="-L$TERMUX_PKG_TMPDIR/_deps -l:libFortranRuntime.a -l:libFortranDecimal.a"
```